### ISSUE TITLE
TrVAE Hyperparameter Tuning

### ISSUE THREAD

**Issue by Cottoneyejoe95** [theislab/scarchesp#5]
We need to find best hyperparameters for trVAE implementation depending on different losses (mse, zinb,nb), 'use_mmd' and depending on dataset criterions (cells, genes,...).

Especially following Parameters:
Layer counts: Neurons, Layers, Latent dim
Loss coefficients: beta, eta, (alpha maybe but with alpha annealing 1 should be good)
Other Params: Dropout-rate, when to use Batchnorms (atm 'on' for zinb and nb, 'off' for mse).

Same for Trainer:
Learning rate, epochs, Learning-rate-decay, Weight decay, Batch_size, Epsilon
